pytest_tests/steps/load.py

```diff
@@ -22,6 +23,25 @@ def get_services_endpoints(
     return [service_config.attributes[endpoint_attribute] for service_config in service_configs]
 
 
+@allure.title("Stop nodes")
+def stop_unused_nodes(hosting: Hosting, used_nodes_count: int):
```

Comment:
If we use new cluster class we can get storage nodes by simply
`cluster.storage_nodes` and this code can be simplified to

```
storage_nodes = cluster.storage_nodes[used_nodes_count:]
    for node in storage_nodes:
        host = node.host
        STOPPED_HOSTS.append(host.config.address)
        host.stop_host("hard")
```

Reply:
done